Translated/Story/08-09 Finding Bruder/ssc_011060.csv

```diff
@@ -0,0 +1,9 @@
+ssc_011060_0010#0,"""Emergency Alert: We have detected strong DOLLS energy readings in the Aerio Region. All ARKS Members, please prepare to sortie."""
+ssc_011060_0020#0,"""Patching through. Can you read me?"""
+ssc_011060_0030#0,"""As you just heard, we've detected a strong DOLLS energy signal. Their target appears to be the ARKS Mining Rigs."""
+ssc_011060_0040#0,"""As its name suggests, it is a type of heavy machinery that we use to excavate resources from underground."""
+ssc_011060_0050#0,"""These excavated resources are necessary for City maintenance as well as the manufacturing of issued ARKS equipment."""
+ssc_011060_0055#0,"""Without these resources, we will not be able to continue fighting the DOLLS."""
+ssc_011060_0060#0,"""Radar equipment is being operated to anticipate DOLLS movement, but it's not uncommon for the radar itself to draw their attention in the same way."""
+ssc_011060_0070#0,"""The rigs are being escorted by resource convoy, but this is a large-scale offensive, so we anticipate a grueling battle."""
```

Comment:
Convoys? Or A convoy?

Reply:
i have absolutely no idea so i fixed it with an english cheat code